pypi_formula_mappings.json

```diff
@@ -740,6 +740,9 @@
     "extra_packages": ["robotframework-archivelibrary", "robotframework-selenium2library", "robotframework-sshlibrary"],
     "exclude_packages": ["certifi", "cryptography"]
   },
+  "rpds-py": {
+    "extra_packages": ["setuptools-rust"]
```

Comment:
Why?

Reply:
because that's what was in #178716, and because it is needed by the build process of rpds-py.

Reply:
(I think this is the kind of indirect dependency @cho-m [explained to me here](https://github.com/Homebrew/homebrew-core/pull/201049#issuecomment-2541635729))